src/js/tracing/reactnavigationv4.ts

```diff
@@ -112,9 +108,8 @@ class ReactNavigationV4Instrumentation extends RoutingInstrumentation {
    * Pass the ref to the app container to register it to the instrumentation
    * @param appContainerRef Ref to an `AppContainer`
    */
-  public registerAppContainer(
-    appContainerRef: AppContainerRef | AppContainerInstance
-  ): void {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public registerAppContainer(appContainerRef: any): void {
```

Comment:
The list of possible types is just not known? 
There isn't a base/common type we could add?

Reply:
As it's a reference to the React Navigation component, the type is pretty complex union of multiple types which is why I felt like it would be tedious to copy over and define inline: https://github.com/react-navigation/react-navigation/blob/ae484782a6ff25ad2864693dfc4694e2b50cf524/packages/core/src/types.tsx#L468-L488